scripts/changelog-pr.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
Well, that's because initially it was just a couple of commands in bash and JS was overkill for that. Would be happy to take this migrated JS script back :)